src/test/java/org/mule/extension/db/integration/model/DerbyTestDatabase.java

```diff
@@ -94,6 +94,13 @@
           "DYNAMIC RESULT SETS 0\n" +
           "EXTERNAL NAME '" + DerbyTestStoredProcedure.class.getName() + ".multiplyInts'";
 
+  public static String SQL_CREATE_SP_CONCATENATE_STRINGS_2 =
```

Comment:
use a declarative name for the test stored procedure.

Reply:
It is not concatenating strings, it is a procedure with returns null in the out parameter of the sp.